docs/ldml/tr35-general.html

```diff
@@ -1361,9 +1368,27 @@ <h2>6 <a name="Unit_Elements" href="#Unit_Elements" id=
 "color: blue">{0} day</span>&lt;/unitName&gt;
   &lt;unitPattern count="other"&gt;<span style=
 "color: blue">{0} days</span>&lt;/unitName&gt;
+&lt;/unit&gt;</pre>
+    <p class='changed'>The german rules are more complicated, because German has both gender and case. They thus have additional information, as illustrated below. Note that if there is no @case attribute, for backwards compatibility the implied case is nominative. The possible values for @case are listed in the <strong>grammaticalFeatures</strong> element. These follow the inheritance specified in Part 1, Section <a href=
```

Comment:
If I go over this in fine detail and notice nits like "german" vs "German", what's the best place to collect feedback and suggestions now?

Reply:
(I see it's live at https://unicode-org.github.io/cldr/ldml/tr35-general.html)